packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -585,6 +585,20 @@ export async function installScriptsAndDependencies(options: {
     throw new Error('Could not locate the package for react-native');
   }
 
+  // We add an exclusionList from metro config. This will be hoisted, but add
+  // an explict dep because we require it directly.
+  const metroConfigPackage = await findPackage('metro-config', {
+    searchPath: rnPackage.path,
```

Comment:
Search path should probably be CWD if we try to require it from the metro config in app package root. Should fail gracefully then